Lib/traceback.py

```diff
@@ -1051,7 +1051,12 @@ def __init__(self, exc_type, exc_value, exc_traceback, *, limit=None,
         # Capture now to permit freeing resources: only complication is in the
         # unofficial API _format_final_exc_line
         self._str = _safe_string(exc_value, 'exception')
-        self.__notes__ = getattr(exc_value, '__notes__', None)
+        notes = None
+        try:
+            notes = getattr(exc_value, '__notes__', None)
+        except Exception:
+            pass
```

Comment:
We don't swallow exceptions like this. What is your use case for defining an exception with a broken ``__getattr__``?

Reply:
I had much the same question on the issue.  Python reporting bugs in user code does not seem like a python bug.  OP needs to justify the python bug claim by quoting doc that says that user exceptions *will* by caught, or make this a feature request, with justification.

Reply:
Thank you for your feedback, I am sorry I did not explain enough. I have tried to add more context to the issue.

About not swallowing exceptions, I don't like it either (and I understand the negative feedback), maybe this patch should be adjusted to at least report that an exception occurred while getting the notes, by doing something like this instead ?

```py
        try:
            self.__notes__ = getattr(exc_value, '__notes__', None)
        except Exception as e:
            self.__notes__ = [
                f'Ignored exception getting __notes__: {_safe_string(e, '__notes__', repr)}']
```

Reply:
Unfortunately we do swallow exceptions like this. `_safe_string()` set a bad precedence since 2823f03a56451f3187a1d85ffcef107f00f6f48e.

Reply:
I have updated the patch to include the "Ignored exception getting __notes__ ..." in the notes as suggested above, from the discussion on the issue it seemed better to include information about the error in that case.